pmlc/rt/jit_executable.cc

```diff
@@ -400,18 +400,20 @@ class JitExecutable final : public Executable {
     }
   }
 
-  void invoke(mlir::ArrayRef<util::BufferPtr> inputBuffers,
-              mlir::ArrayRef<util::BufferPtr> outputBuffers) final {
+  double invoke(mlir::ArrayRef<util::BufferPtr> inputBuffers,
+                mlir::ArrayRef<util::BufferPtr> outputBuffers) final {
+    bindArguments(inputBuffers, outputBuffers);
```

Comment:
This should be included in the stopwatch time.

Reply:
Done.